scale/job/configuration/interface/job_interface.py

```diff
@@ -88,6 +97,26 @@
                 },
             },
         },
+        'mount': {
+            'type': 'object',
+            'required': ['name', 'path'],
+            'additionalProperties': False,
+            'properties': {
+                'name': {
+                    'type': 'string',
```

Comment:
Yes, we should, but that can happen in another issue as we need to go back and apply it across all of the schemas and check to make sure all docs state the same thing.